core/src/main/res/values/strings.xml

```diff
@@ -178,6 +178,11 @@
         <item quantity="one">%d episode deleted.</item>
         <item quantity="other">%d episodes deleted.</item>
     </plurals>
+    <plurals name="deleted_multi_episode_batch_label">
+        <item quantity="zero">%d episodes selected, %d deleted. Only downloaded files removed.</item>
+        <item quantity="one">%d episodes selected, %d deleted. Only downloaded files removed.</item>
+        <item quantity="other">%d episodes selected, %d deleted. Only downloaded files removed.</item>
```

Comment:
@kunes could you help with the wording?

Reply:
How about `%d episodes selected, %d downloads deleted.`?